src/index.html

```diff
@@ -1,14 +1,83 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
-</html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+    content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet"
+    href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+  </head>
+
+  <body>
+
+    <header class="header">
+      <a href="#"
+      class="logo">
+        <img src="images/logo.png"
+        alt="logo-moyo">
```

Comment:
преглянь код і пофікси відступи для атррибутів
```suggestion
      <a href="#" class="logo">
        <img src="images/logo.png" alt="logo-moyo">
```